.changeset/small-bugs-prove.md

```diff
@@ -0,0 +1,34 @@
+---
+'astro': minor
+---
+
+Order Astro styles last, to override imported styles
+
+This fixes CSS ordering so that imported styles are placed *higher* than page/component level styles. This means that if you do:
+
+```astro
+---
+import '../styles/global.css';
+---
+<style>
+  body {
+    background: limegreen;
+  }
+</style>
+```
+
+The `<style>` defined in this component will be placed *below* the imported CSS. When compiled for production this will result in something like this:
+
+```css
+/* /src/styles/global.css */
```

Comment:
is this demo describing that the output might look like one css file with rules sources from two files? updating the comments to reflect that might be clearer

Reply:
It's describing the approximate order output, with imported styles being placed before Astro component styles.